python/rikai/types/vision.py

```diff
@@ -119,7 +116,8 @@ def display(self, **kwargs):
         """
         from IPython.display import Image
 
-        return Image(self.uri, **kwargs)
```

Comment:
Curious why many libraries don't support loading from HTTPs out of the box (Jupyter, TensorFlow, others?). Jupyter appears to support http but not https?

Reply:
Usually a good design should design against to a "file-object" like interface, i'd say.